gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsCommonTrait.scala

```diff
@@ -48,10 +49,16 @@ trait GlutenTestsCommonTrait
     status
   }
 
+  protected def testWithGlutenPrefix(testName: String, testTag: Tag*)(testFun: => Any)(implicit
+      pos: Position): Unit = {
+    test(GLUTEN_TEST + testName, testTag: _*)(testFun)
+  }
   override protected def test(testName: String, testTags: Tag*)(testFun: => Any)(implicit
       pos: Position): Unit = {
     if (shouldRun(testName)) {
       super.test(testName, testTags: _*)(testFun)
+    } else {
+      logInfo(s"Ignore test case: $testName")
```

Comment:
@Yohahaha, did you mean we can somehow get a summarized report for all ignored tests? If so, there may be no need to log ignored test here.

Reply:
I'd expect such way.
```scala
if (shouldRun(testName)) {
      super.test(testName, testTags: _*)(testFun)
    } else {
      super.ignore(testName, testTags: _*)(testFun)
    }
```